src/imports/logger/index.js

```diff
@@ -1,27 +1,19 @@
+// libraries
 import winston from 'winston'
```

Comment:
@sirpy has asked do not use folder/index.js imports

please, move the logger back to the src/imports/logger.js